color_coder_test.py

```diff
@@ -0,0 +1,21 @@
+import colorpair_generator as color
+
+
+def test_colorpair_for_pairnumber(pair_number,
```

Comment:
Try including the expectation in the test name. See if more test possibilities come out.